### DISCUSSION TITLE
using x-slot inside livewire component

### DISCUSSION THREAD

**Discussion by foremindnet** [livewire/livewire#4294] [1. Help]
Not sure if this is a bug, feature request or i've just not used this correctly. I am trying to use x-slot inside of a <livewire> tag like below but they dont seem to be passing to the livewire component.

`
<livewire:components.tabs>
<x-slot name="tab-one">First tab</x-slot>
<x-slot name="tab-two">Second tab</x-slot>
</livewire:components.tabs>
`

**Comment by joshhanley** [comment]
@sihq Livewire doesn't support slots at the moment. Have a look at this discussion https://github.com/livewire/livewire/discussions/4199 for more details.

Hope this helps!
